docs/ecosystem/tools.md

```diff
@@ -1,10 +1,10 @@
 # Third-party tooling
 
-## inlang
+## Sherlock – i18n inspector (VS Code extension)
 
-inlang is the ecosystem to adapt and expand software products to new markets, and that has [VSCode extensions](https://marketplace.visualstudio.com/items?itemName=inlang.vs-code-extension).
+Sherlock is a VS Code extension that helps you to extract, edit & inspect your i18n keys in your codebase. It is available in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=inlang.sherlock).
```

Comment:
`https://marketplace.visualstudio.com/items?itemName=inlang.sherlock` is dead link
Where would it?